packages/atomic-react/README.md

```diff
@@ -10,6 +10,13 @@ The integration of React-based projects (using JSX) with Web based components ca
 
 `npm i @coveo/atomic-react`
 
+### Alternate installation -- CDN
+
+The library is also available from Coveo's CDN as an `IIFE` (Immediately Invoked Function Expression). This is an approach that we recommend only for quick prototyping and testing, and that can be used with very minimal front end tooling and build tools.
+
+The `IIFE` approach in production is discouraged - the browser has to download the entire library code, regardless of which components are actually used. Using a bundler (for example Webpack) is recommended.
+
+You can read more on this approach down below.
```

Comment:
```suggestion
You can read more about this approach [below](#usage-with-cdn-scripts--iife-).
```
(not totally sure about the link though, and it depends on you accepting my suggestion below 😉)